### PR TITLE
radare2 2.0.1

### PR DESCRIPTION
**Pull request by makigumo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/radare2.rb [hunk 1]
```diff
@@ -82,6 +89,10 @@ def install
     end
     system "make", "install"
 
+    # remove leftofer symlinks
```
Comment (ilovezfs) [inline Formula/radare2.rb]: has this been reported

**Comment by makigumo** [comment]
Not sure what the build problem is. But the build before went fine (https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/10962/) and only a line comment was added.

**Comment by ilovezfs** [comment]
Thanks @makigumo! Shipped.

**Comment by ilovezfs** [comment]
parallelization issue reported here https://github.com/radare/radare2/issues/8704

**Pull request closed without merging**